acceptance/provider/synced_folder_spec.rb

```diff
@@ -29,5 +29,12 @@
     status("Test: doesn't mount a disabled folder")
     result = execute("vagrant", "ssh", "-c", "test -d /foo")
     expect(result.exit_code).to eql(1)
+
+    status("Test: persists a sync folder after a manual reboot")
+    result = execute("vagrant", "ssh", "-c", "sudo reboot")
+    expect(result).to exit_with(255)
+    result = execute("vagrant", "ssh", "-c", "cat /vagrant/foo")
```

Comment:
Should there be some kind of wait here while the guest reboots? :thinking:

Reply:
It doesn't seem necessary -- the wait happens automatically.